rust/perspective-python/package.json

```diff
@@ -12,9 +12,10 @@
     "scripts": {
         "build": "node build.mjs",
         "clean": "rimraf perspective.data && rimraf dist && rimraf build",
-        "test": "python -m pytest perspective/tests"
+        "test": "node test.mjs"
     },
     "devDependencies": {
+        "@finos/perspective-scripts": "workspace:*",
```

Comment:
Posthumous review update -

Packages in the `tools/` directory are private to the source repository and they can't be included as dependencies in the published packages, even if they are just dev dependencies, because these tools ultimate are intended to depend on Perspective also and you end up with circular imports.

Reply:
Makes sense. I didn't know that we don't publish `@finos/perspective-scripts` to NPM